charts/cht-user-management/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: base-single-app
-  repository: file://../base-single-app
+  repository: https://docs.communityhealthtoolkit.org/helm-charts
   version: 0.2.0
-digest: sha256:0a0341817c9308b919ddd591db8df326b60cee362ea584e9e87c1195e9ab200b
-generated: "2024-01-11T13:57:53.726465+01:00"
+digest: sha256:284a3dc5e9eb4c400a3abff6f49faa047ad11d2dbbc873e8adf8829cde321aae
```

Comment:
While the checksum is different the file-content is not, so no release is needed:
```
❯ md5sum ** | sort
01e47ac671716bb543995cc975a5c9c0  file/templates/deployment.yaml
01e47ac671716bb543995cc975a5c9c0  repo/templates/deployment.yaml
1bfe71634802573c77a389533565373e  file/templates/ingress.yaml
1bfe71634802573c77a389533565373e  repo/templates/ingress.yaml
220d64b3887fb4626fdecd1f098846dc  file/templates/NOTES.txt
220d64b3887fb4626fdecd1f098846dc  repo/templates/NOTES.txt
2608eac7e2bdc982744feaec6aa3afed  file/values.yaml
2608eac7e2bdc982744feaec6aa3afed  repo/values.yaml
2ab44ceb08e5e2dbee62c6e2ef4a837f  file/templates/tests/test-connection.yaml
2ab44ceb08e5e2dbee62c6e2ef4a837f  repo/templates/tests/test-connection.yaml
46ae89b0fa33a84e88570b5ee6c97a0d  file/templates/service.yaml
46ae89b0fa33a84e88570b5ee6c97a0d  repo/templates/service.yaml
8c3f6a85b23d077a5ad5fa9b7a1c34de  file/Chart.yaml
8c3f6a85b23d077a5ad5fa9b7a1c34de  repo/Chart.yaml
9c2e6966bf56986985ecb276b00e2d1e  file/templates/_helpers.tpl
9c2e6966bf56986985ecb276b00e2d1e  repo/templates/_helpers.tpl
```